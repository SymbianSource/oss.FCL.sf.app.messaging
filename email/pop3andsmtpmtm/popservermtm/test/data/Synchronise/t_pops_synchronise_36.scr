+OK demon POP3 server ready (demon.co.uk)
+OK 
+OK 
+OK 20 30387
+OK 20 messages
1 1519
2 1558
3 1599
4 1559
5 1355
6 1519
7 1519
8 1598
9 1557
10 1598
11 1519
12 1597
13 1558
14 1312
15 1476
16 1477
17 1557
18 1516
19 1517
20 1477
.
+OK 20 messages
1 t_pops_synchronise:message026
2 t_pops_synchronise:message027
3 t_pops_synchronise:message028
4 t_pops_synchronise:message029
5 t_pops_synchronise:message030
6 t_pops_synchronise:message031
7 t_pops_synchronise:message032
8 t_pops_synchronise:message033
9 t_pops_synchronise:message034
10 t_pops_synchronise:message035
11 t_pops_synchronise:message036
12 t_pops_synchronise:message038
13 t_pops_synchronise:message039
14 t_pops_synchronise:message040
15 t_pops_synchronise:message041
16 t_pops_synchronise:message042
17 t_pops_synchronise:message043
18 t_pops_synchronise:message044
19 t_pops_synchronise:message045
20 t_pops_synchronise:message046
.
-ERR Unknown Command "CAPA"
+OK 
Return-Path: <Andrew.Fawcett@teleca.com>
Received: from punt-2.mail.demon.net by mailstore for freddy@yashar.demon.co.uk
          id t_pops_synchronise:message046; Mon, 20 Aug 2001 17:32:14 GMT
Received: from mail.teleca.com ([195.166.58.131]) by punt-2.mail.demon.net
           id aa2026233; 20 Aug 2001 17:32 GMT
Received: from teleca126.teleca.com (unverified) by teleca161.teleca.com
 (Content Technologies SMTPRS 4.1.5) with ESMTP id <Tac100102557da8ad42@teleca161.teleca.com> for <freddy
@yashar.demon.co.uk>;
 Mon, 20 Aug 2001 18:33:46 +0100
Received: by teleca126 with Internet Mail Service (5.5.2650.21)
        id <Q44VHMHM>; Mon, 20 Aug 2001 18:37:02 +0100
Message-ID: <t_pops_synchronise_message046@JANUS.teleca.com>
From: Andrew Fawcett <Andrew.Fawcett@teleca.com>
To: 'freddy' <freddy@yashar.demon.co.uk>
Subject: Message 46
Date: Mon, 20 Aug 2001 18:28:48 +0100
Return-Receipt-To: Andrew Fawcett <Andrew.Fawcett@teleca.com>
MIME-Version: 1.0
X-Mailer: Internet Mail Service (5.5.2650.21)
Content-Type: text/plain;
        charset="iso-8859-1"

Forty-Six Forty-Six Forty-Six Forty-Six Forty-Six Forty-Six Forty-Six
.
** Read cancelled
+OK 
