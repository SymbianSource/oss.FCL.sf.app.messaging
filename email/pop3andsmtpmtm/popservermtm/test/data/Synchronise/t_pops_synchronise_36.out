STAT
LIST
UIDL
CAPA
TOP 20 1
QUIT
